src/Tribe/Rewrite.php

```diff
@@ -731,7 +735,9 @@ public function get_localized_bases( array $bases, array $domains ) {
 			return $cached;
 		}
 
-		$localized_bases = tribe( 'tec.i18n' )->get_i18n_strings_for_domains( $bases, [ $locale ], $domains );
+		$flags = I18n::COMPILE_STRTOLOWER;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 11 spaces but found 1 space

<sub>hash: 154b39f3e54764543e7914bb99b54d74</sub>